examples/convert_mri_and_trans.py

```diff
@@ -0,0 +1,168 @@
+"""
+=======================================================================
+Save a T1 weighted MRI scan to BIDS, and encode a transformation matrix
```

Comment:
I think the "transformation matrix" part makes it somewhat less accessible to a new user. How about:

"Save and load T1-weighted MRI scan along with anatomical landmark coordinates in BIDS"

Reply:
agreed, but shortened to `Save and load T1-weighted MRI scan along with anatomical landmarks in BIDS` so that it fits without lots of `# noqa: E501`